packages/massa-proto-cli/src/ts-gen.ts

```diff
@@ -0,0 +1,19 @@
+import { ProtoFile } from './protobuf';
+
+/**
+ * Generates the typescript code to import the helper file
+ *
+ * @param protoFile - The proto file data
+ * @param helperRelativePath - The relative path to the previously generated helper
+ *
+ * @returns The typescript code to import the helper and the Account interface
+ */
+function generateImports(protoFile: ProtoFile, helperRelativePath): string {
```

Comment:
Your not only importing thing, but also exporting the interface. You should remove that part of the code in another function or update the function name if you want to do both. The second approach is messier becasue functionaly you are doing two different things